es/12_1_Usando_Script_Condicionales.md

```diff
@@ -84,13 +84,13 @@ ENDIF
  OP_CHECKSIG
 ```
 
-La declaración que se evaluará como "Verdadero" o "Falso" se coloca en la pila _ antes de ejecutar el "IF", luego se ejecuta el bloque de código correcto en función de ese resultado.
+La declaración que se evaluará como "Verdadero" o "Falso" se coloca en la _pila_ antes de ejecutar el "IF", luego se ejecuta el bloque de código correcto en función de ese resultado.
 
 Este código de ejemplo en particular está destinado a ser una firma múltiple 1 de 2 de un pobre hombre. 
 El propietario de `<privKeyA>` pondría `<signatureA> <pubKeyA> True` en su script de desbloqueo, mientras que el propietario de` <privKeyB> `pondría` <signatureB> <pubKeyB> False` en su script de desbloqueo. 
 Ese final de "Verdadero" o "Falso" es lo que marca la declaración "IF" / "ELSE". Le dice al script con qué hash de clave pública se debe verificar, luego el `OP_EQUALVERIFY` y el` OP_CHECKSIG` al final hacen el trabajo real.
```

Comment:
- "IF" / "ELSE" was `IF` / `ELSE` (written as `` `IF` / `ELSE` ``) in the original text.
- ` OP_CHECKSIG` is enclosing a white space at the beginning: `OP_CHECKSIG` is a cleaner form.